package.json

```diff
@@ -31,5 +31,17 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "resolutions": {
+        "mem": "^5.1.1",
+        "lodash": "^4.17.15",
+        "stringstream": "^1.0.0",
+        "js-yaml": "^3.13.1",
+        "just-extend": "^4.0.2",
+        "extend": "^2.0.2",
+        "merge": "^1.2.1",
+        "randomatic": "^3.1.1",
```

Comment:
The jump from `1.1.3` is likely to break things. Can we fix this by bumping `babel-cli` instead?

Chain: `babel-cli` -> `chokidar` -> `anymatch` -> `micromatch` -> `braces` -> `expand-range` -> `fill-range` -> `randomatic`